docs/source/download.rst

```diff
@@ -110,6 +110,10 @@ examples on the :doc:`documentation page <documentation>`.
 Git instructions for developers
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Cython >= 0.17 is required to build Firedrake. Install it using pip ::
+
+ pip install "Cython>=0.17"
```

Comment:
I was going to set up environments to test which version worked, but hadn't got around to it yet. Should we just change the document to recommend 0.20 and leave it at that?